service/controller/aws/v16/resource/clusterconfigmap/current.go

```diff
@@ -0,0 +1,38 @@
+package clusterconfigmap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/v16/key"
+	awskey "github.com/giantswarm/cluster-operator/service/controller/aws/v16/key"
+)
+
+func (r *StateGetter) GetCurrentState(ctx context.Context, obj interface{}) ([]*v1.ConfigMap, error) {
+	customObject, err := awskey.ToCustomObject(obj)
```

Comment:
```suggestion
	cr, err := awskey.ToCustomObject(obj)
```

Like the kubeconfig resource we need this long term. So we should follow the latest naming conventions.